tests/Feature/CheckinTest.php

```diff
@@ -441,7 +445,8 @@ public function testCheckinAtBus603Potsdam() {
                              'departure'   => $trip['stopovers'][0]['departure'],
                              // Reiterweg ist 6 Stationen hinter Schloss Cecilienhof
                              'destination' => $trip['stopovers'][5]['stop']['id'], // Reiterweg
-                             'arrival'     => $trip['stopovers'][5]['arrival']
+                             'arrival'     => $trip['stopovers'][5]['arrival'],
+                             'checkinVisibility' => "0"
```

Comment:
string or integer?

Reply:
...and format please

Reply:
https://github.com/Traewelling/traewelling/pull/410#discussion_r647724103

Reply:
https://github.com/Traewelling/traewelling/pull/410#discussion_r647729264